rtos/TARGET_CORTEX/mbed_rtx_handlers.c

```diff
@@ -107,29 +107,41 @@ static const char *error_msg(int32_t status)
     }
 }
 
+static void trap_rtx_error(int32_t rtx_status, mbed_error_status_t error_status)
```

Comment:
Minor comment: For readability, it would be better if we can switch the order of args in trap_rtx_erro() to be inline with how MBED_ERROR1 uses it.

Reply:
The ordering is intentional, as a micro-size optimisation. Seeing as it's not public API, we can do what we want, so it makes sense to make the prototype of `trap_rtx_error` as close as possibel to that of `EvrRtxXXX` to avoid the need to put argument-shuffling MOV instructions in each function.